plugins/woocommerce/tests/php/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -55,7 +55,7 @@ public function test_product_count_visibility_with_empty_cart() {
 		// Tests badge is not shown, because product count is not greater than zero when "greater_than_zero" is selected.
 		$block  = parse_blocks( '<!-- wp:woocommerce/mini-cart {"productCountVisibility":"greater_than_zero"} /-->' );
 		$output = render_block( $block[0] );
-		$this->assertStringNotContainsString( '<span class="wc-block-mini-cart__badge"', $output );
+		$this->assertStringContainsString( '<span class="wc-block-mini-cart__badge"', $output );
```

Comment:
I changed this behaviour in the PR to hide empty badge with CSS so the initial state from local storage has an element to render into if the page is cached.